doc/sphinx-guides/source/installation/config.rst

```diff
@@ -501,6 +501,14 @@ A Dataverse installation can alternately store files in a Swift or S3-compatible
 
 A Dataverse installation may also be configured to reference some files (e.g. large and/or sensitive data) stored in a web-accessible trusted remote store.
 
+A Dataverse installation can be configured to allow out of band upload by setting the ``dataverse.files.\<id\>.upload-out-of-band`` JVM option to ``true``.
+This option allows API users to add files with the :ref:`add-file-api` call. It also allows API users to register the metadata of a file with the :ref:`add-file-metadata-api` call for a file that was uploaded directly to the storage.
```

Comment:
Isn't add-file-api already enabled? i.e. "By default, Dataverse support uploading files via the add-file-api. With S3 stores, a direct upload process can be enabled to allow sending the file directly to the S3 store (without any intermediate copies on the Dataverse server). With the upload-out-of-band option enabled, it is also possible for file upload to be managed manually or via third-party tools, with the add-file-metadata-api call used to add metadata and inform Dataverse that a new file has been added to the relevant store."

Reply:
I have reread the relevant documentation, and it makes more sense now. I can also see why it is the same option for the UI as API, especially since the new UI will use the API direct upload flow for the upload-redirects (if the current UI does not yet use that).